packages/ubuntu_wsl_setup/test/named_event_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:ubuntu_wsl_setup/services/named_event.dart';
+
+void main() {
+  const methodChannel = MethodChannel(NamedEventConstants.channel);
+  TestWidgetsFlutterBinding.ensureInitialized();
+  final binding =
```

Comment:
s/binding/messenger/

Reply:
> I'm struggling with that one.

Sorry for the confusion. I meant just the variable name because it's not the binding but a messenger. :)

> `TestWidgetsFlutterMessenger` doesn't exist. sweat_smile

Hmm, it's defined in flutter_test. Did you have the words in the correct order? :smile: 

> `TestDefaultBinaryMessengerBinding.instance.initInstances();` is not what I'm looking for because instance can be null. `initInstances()` is not static.
> 
> What is the correct call, then?

Not sure what you prefer but we could even combine it into one line since we're specifically interested in the messenger:

```dart
final messenger =
      TestWidgetsFlutterBinding.ensureInitialized().defaultBinaryMessenger;
```

Reply:
I never remember how to spell TestWidgetsFlutterBinding, it's always try-n-error with the auto-completer... :smile:

Reply:
Oops, sorry! Sed :smile: